src/utils.cpp

```diff
@@ -104,10 +104,10 @@ String implode(const Vector<String>& vec, const char delimiter /* = ' ' */) {
 String& trim(String& str) {
   // Trim front
   str.erase(str.begin(),
-            std::find_if(str.begin(), str.end(), std::not1(std::ptr_fun<int, int>(::isspace))));
+            std::find_if(str.begin(), str.end(), [](unsigned char c) { return !::isspace(c); }));
```

Comment:
Lambdas are a C++11 feature. We currently support older compilers that do not support C++11 features.

Reply:
I understand. I don't have a C++03 compiler at hand, so I'll close the PR. Thanks for the review either way! :)